### DISCUSSION TITLE
Gnoland way to solve people problems

### DISCUSSION THREAD

**Discussion by felipebrunet** [gnolang/gno#2849] [General]
As my current work is related to creating value for customers, I wonder how is Gnoland going to provide value to people. By value I mean satisfaction to their needs and problems, not money.

I see 3 types of attributes in Gno (my guess is):
1. Related to currency:
- Borderless payment.
- Censorship resistant.
- Pseudo anonymous payment.
- Somewhat an inflation resistant store of value.

2. Related to  information (my guess is):
- Censorship resistant communication.
- Decentralized storage of information.

3. Related to customer experience and UX (my guess is):
- Reducing number of passwords used (for example authenticating via Adena wallet)
- Fast and simple web payments (for example  with Adena wallet).

However great and altruistic are attributes (1) and (2), I find difficult to find who exactly are the People (their location, characteristics and number of individuals) that are needing these attributes to satisfy their needs.

Maybe people in authoritarian places need the attributes of (2), e.g. Brazil, which censored Twitter, I guess.

In my opinion, Bitcoin was made because of its attributes, not so much the specific target users it would be useful to. I see it having attributes (1) and I still fail to see how many people really have that problem. 

I believe attributes (3) can help many people in their needs, although I know that UI needs  are less fundamental than freedom and rights needs. However, given the state of the world, attributes (1) and (2) may have a large group of users in need.

I will look further this way, finding people with a problem and then try to find a solution that Gno could provide.

Have a nice day

**Comment by thehowl** [comment]
There are a variety of objectives with what we want to achieve with gno.land; and in some ways, they are also kind of in flux. I'll post a few references you can take a look at for better understanding.

- Creating better smart contracting.
  - [This talk](https://www.youtube.com/watch?v=betUkghf_jo) by Jae does some comparisons with other languages, and talks about the benefits of Gno more in general.
- Creating better ways to discuss, talk and interact with others (censorship-resistance, yes, but that's not the only point).
  - [This talk](https://www.youtube.com/watch?v=m2zVj-BxRPY) by Jae talks about some of the points related to this. Gno.land is a space to create a reddit clone and get set up in a few hours; not just a place for making de-fi applications.

We'll be working on our messaging to try to partly address in written form part of what you ask here. But it's complex to come back with a satisfactory, written answer. :)

**Reply by felipebrunet** [reply]
Hey thanks. I did not watch the first Jae talk you mention, I'll check it up.
I still wonder what are the specific users that will value those 2 objectives. I can see defi users and people from censor-prone territories getting value from Gnoland.
But what about non tech people? Maybe they will not be direct users, but they can be indirect users.
Gnoland tech may be wrapped by some company(s) that will value and embrace its transparency and auditability, and would like to provide a secure defi/social-network product.
If that is the case, my guess is that people will be willing to pay (in traditional fiat of course) for the value they are getting. 
But (for example) if a company runs its whole independent Gno chain, how will that be decentralized? Will regular users reject that centralized model even though they might receive value?
How do you engage non technical users with the condition of embracing core values (like decentralization) but they may not consider it a problem?
Cheers!
